### PR TITLE
TimeoutSampler: improve print for lambda calls

### PR DESCRIPTION
**Pull request by myakove**
##### Short description:
Print all args when function is lambda

### CODE REVIEW

File: ocp_resources/utils.py [hunk 1]
```diff
@@ -153,7 +153,12 @@ def _get_func_info(self, _func, type_):
                     return f"{res}.{_func.__qualname__.split('.')[1]}"
 
                 elif type_ == "__name__":
-                    return f"lambda: {'.'.join(_func.__code__.co_names)}"
+                    _free_vars = ""
+                    free_vars = _func.__code__.co_freevars
+                    if free_vars:
+                        _free_vars = f"{'.'.join(free_vars)}."
```
Comment (rnetser) [inline ocp_resources/utils.py:159]: free_vars = _func.__code__.co_freevars
 _free_vars = f"{'.'.join(free_vars)}." if free_vars else ""

Comment (myakove) [reply ocp_resources/utils.py]: ACK

**Comment by myakove** [comment]
```
ocp_resources.utils 2021-09-12 06:33:01 INFO Waiting for 30 seconds, retry every 1 seconds. (Function: ocp_resources.resource.wait_for_conditions.lambda: self.instance.status.conditions)
```

**Pull request merged**